src/main/java/com/ncedu/fooddelivery/api/v1/controllers/AboutController.java

```diff
@@ -15,6 +17,7 @@
 
     @GetMapping("/api/v1/about")
     public AboutDTO about() {
+        log.info("GET /api/v1/about calling.");
```

Comment:
Do we need all this logs with INFO level?

Reply:
I think this ok on this stage. But I can replace 'info' to 'debug' level, it's will be ok?